templates/aws/outposts.html

```diff
@@ -0,0 +1,202 @@
+    </div>
+  </div>
+  <div class="row u-sv2">
+    <div class="col-12">
+      <h2>Get Ubuntu Pro on AWS Outposts now:</h2>
+    </div>
+  </div>
+  <div class="row p-divider">
+    <div class="col-3 p-divider__block">
+      <h3 class="p-heading--four"><a href="https://aws.amazon.com/marketplace/pp/B087L1R4G4" class="p-link--external">20.04 LTS</a></h3>
+    </div>
+    <div class="col-3 p-divider__block">
+      <h3 class="p-heading--four"><a href="https://aws.amazon.com/marketplace/pp/B0821T9RL2" class="p-link--external">18.04 LTS</a></h3>
+    </div>
+    <div class="col-3 p-divider__block">
+      <h3 class="p-heading--four"><a href="https://aws.amazon.com/marketplace/pp/B0821T9RL2" class="p-link--external">18.04 FIPS LTS</a></h3>
+    </div>
+    <div class="col-3 p-divider__block">
+      <h3 class="p-heading--four"><a href="https://aws.amazon.com/marketplace/pp/B0821WW873" class="p-link--external">16.04 LTS</a></h3>
+    </div>
+    <div class="col-3 p-divider__block">
+      <h3 class="p-heading--four"><a href="https://aws.amazon.com/marketplace/pp/B0821WW873" class="p-link--external">16.04 FIPS LTS</a></h3>
+    </div>
+    <div class="col-3 p-divider__block">
+      <h3 class="p-heading--four"><a href="https://aws.amazon.com/marketplace/pp/B0821V7QJP" class="p-link--external">14.04 LTS</a></h3>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <p class="u-no-margin--bottom"><strong>Read more about</strong> <a href="/aws/pro">Ubuntu Pro&nbsp;&rsaquo;</a></p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2 class="u-sv2">Launching Ubuntu on AWS Outposts</h2>
+    <ol class="p-stepped-list">
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title">
+          Select the region your AWS Outpost is attached
+        </h3>
+        <div class="p-stepped-list__content">
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/78ea4b05-aws-outposts-regions.png",
+              alt="",
+              width="300",
+              height="150",
+              hi_def=True,
+              loading="auto",
+            ) | safe
+          }}
+        </div>
+      </li>
+    
+      <li class="p-stepped-list__item">
+        <h3 class="p-stepped-list__title">
+          Select the Ubuntu version you want to launch
+        </h3>
+        <div class="p-stepped-list__content">
+        <p>Launch an EC2 instance and select Ubuntu from the instance launch quickstart, or visit the marketplace for more Ubuntu options.</p>
+          {{
+            image(
+              url="https://assets.ubuntu.com/v1/03238ea8-aws-outputs-ubuntu-image.png",
+              alt="",
+              width="940",
+              height="170",
+              hi_def=True,
+              loading="auto",
+              attrs={"class": "u-hide--small;"}
```

Comment:
```suggestion
              attrs={"class": "u-hide--small"}
```
my bad - no semicolon